Source/WebCore/platform/graphics/SystemFallbackFontCache.cpp

```diff
@@ -46,7 +46,7 @@ SystemFallbackFontCache& SystemFallbackFontCache::forCurrentThread()
 
 SystemFallbackFontCache* SystemFallbackFontCache::forCurrentThreadIfExists()
 {
-    auto* cache = FontCache::forCurrentThreadIfExists();
+    const CheckedPtr cache = FontCache::forCurrentThreadIfExists();
```

Comment:
Just `CheckedPtr`. We don't normally use const for local variables, nor does it help with static analysis.